blocks/columns/columns.css

```diff
@@ -19,16 +15,96 @@
   display: block;
 }
 
-@media (width >= 1024px) {
+.columns-divider-variant > div {
+  display: flex;
+  gap: 20px;
+
+  > div {
+    order: 1;
+  }
+
+  > div:first-child {
+    flex-basis: 33%;
+    font-size: var(--body-font-size-xl);
+    margin-bottom: 0;
+
+        p {
+          font-size: var(--body-font-size-xl);
+          text-align: left;
+          color: var(--neutrals-100);
+        }
+  }
+
+  > div:last-child {
+    flex-basis: 66%;
+    border-left: 1px solid var(--neutrals-20);
+    padding-left: 20px;
+    color: var(--neutrals-100);
+    font-size: var(--body-font-size);
+  }
+}
+
+@media (width >= 900px) {
   .columns > div {
-    align-items: center;
-    flex-direction: unset;
+    align-items: start;
+    flex-direction: row;
     gap: 32px;
+
+    > div {
+      flex: 1;
+      order: unset;
+    }
   }
+}
+
+@media (width >= 768px) and (width <= 1024px) {
```

Comment:
Can we do `media (width <= 1024px)` and then below `media(width <= 768px)` separately?